src/pages/settings/Wallet/ChooseTransferAccountPage.tsx

```diff
@@ -1,31 +1,33 @@
-import React from 'react';
+import React, {useMemo} from 'react';
 import type {GestureResponderEvent} from 'react-native';
 import {View} from 'react-native';
-import type {OnyxEntry} from 'react-native-onyx';
-import {withOnyx} from 'react-native-onyx';
+import {useOnyx} from 'react-native-onyx';
+import FullscreenLoadingIndicator from '@components/FullscreenLoadingIndicator';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import Icon from '@components/Icon';
+import getBankIcon from '@components/Icon/BankIcons';
 import * as Expensicons from '@components/Icon/Expensicons';
 import MenuItem from '@components/MenuItem';
 import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
+import {getLastFourDigits} from '@libs/BankAccountUtils';
 import Navigation from '@libs/Navigation/Navigation';
 import * as BankAccounts from '@userActions/BankAccounts';
 import * as PaymentMethods from '@userActions/PaymentMethods';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
-import type {AccountData, WalletTransfer} from '@src/types/onyx';
-import PaymentMethodList from './PaymentMethodList';
+import type {AccountData} from '@src/types/onyx';
+import type {BankName} from '@src/types/onyx/Bank';
+import isLoadingOnyxValue from '@src/types/utils/isLoadingOnyxValue';
 
-type ChooseTransferAccountPageOnyxProps = {
-    /** Wallet transfer propTypes */
-    walletTransfer: OnyxEntry<WalletTransfer>;
-};
+function ChooseTransferAccountPage() {
+    const [walletTransfer, walletTransferResult] = useOnyx(ONYXKEYS.WALLET_TRANSFER);
```

Comment:
In the previous code, I see we set the default value for this field is {}. Should we add a default value here?

```suggestion
        const [walletTransfer, walletTransferResult] = useOnyx(ONYXKEYS.WALLET_TRANSFER, {initialValue: {}});
```

Reply:
Yeah, Look fine. I ask you because this is a new implementation that doesn't exist before